linker/packages/linker/src/WESLGrammar.ts

```diff
@@ -432,22 +422,10 @@ const variable_or_value_statement = or(
 const variable_updating_statement = or(
   seq(
     lhs_expression,
-    or(
-      "=",
-      "<<=", // TODO could this be handled with a lexer rule?
-      ">>=",
-      "%=",
-      "&=",
-      "*=",
-      "+=",
-      "-=",
-      "/=",
-      "^=",
-      "|=",
-    ),
+    or("=", "<<=", ">>=", "%=", "&=", "*=", "+=", "-=", "/=", "^=", "|="),
```

Comment:
WDYT about a lexer match instead of a parser match?

Reply:
I think we could do that. The main reason why I didn't is that it already took me long enough to figure out the whole `<` and `<<=` ambiguity part.